src/proj.rs

```diff
@@ -1018,7 +1018,11 @@ impl convert::TryFrom<(&str, &str)> for Proj {
     }
 }
 
-struct PjInfo {
+/// Info about the current PROJ definition
+///
+/// [PROJ reference documentation](https://proj.org/development/reference/datatypes.html?highlight=has_inverse#c.PJ_PROJ_INFO)
+#[derive(Clone, Debug)]
+pub struct PjInfo {
```

Comment:
```suggestion
pub struct ProjInfo {
```

Since this was private, we can freely change the name. I think it's nice to match the upstream name, or at least idiomatically translate them.

This is completely subjective, so feel free to take it or leave it.

Reply:
ahah oh darn, good point @frewsxcv.

With perfect hindsight, maybe an idiomatic conversion would map `PJ_INFO` to `proj::Info` and PJ_PROJ_INFO to `proj::ProjInfo`

But, I'd understand if doing that whole swap is more work/confusion than it's worth.

Reply:
Ah yes, and I've broken the build. Swapping will be a breaking change, won't it? (Not necessarily a reason not to do it…)

Reply:
[6f89e01](https://github.com/georust/proj/pull/133/commits/6f89e019a650c76a4f4bb482e27548480a4eb18b) 👀 

Oh no. 😆 

Well @urschrei, if you want to see this through:
```diff
- pub struct Projinfo {
+ pub struct Info {
...
  }

+ #[deprecated(message = "renamed to Info")]
+ pub type Projinfo = Info
```

```

Reply:
>  Swapping will be a breaking change, won't it?

I don't *think* so? I __think__ you'd just need to correspondingly rename the (here-to-fore private) usages of PjInfo.

Reply:
We can't use `Info` because we have a public trait named `Info`: https://github.com/georust/proj/blob/main/src/proj.rs#L231

😭

Anyone got a good suggestion for an alternative name?

Reply:
oh my gosh what a pjckle. I surrender. 🏳️

Reply:
Ok wait, I think I have a proposal. One minute.

Reply:
Proposal: https://github.com/georust/proj/pull/134

I think it puts us in a nice place, but it's got some downsides - I know you're no fan of macros, and it's potentially confusing to replace a trait with a struct of the same name, so no hard feelings if you'd prefer to just leave the names as they are.